.github/workflows/build_and_test.yml

```
@@ -174,6 +174,26 @@ jobs:
           submodules: true
           fetch-depth: 1
 
+      - name: Check Postgres submodules revision
+        run: |
+          # This is a temporary solution to ensure that the Postgres submodules revision is correct (i.e. the updated intentionally).
+          # Eventually it will be replaced by a regression test https://github.com/neondatabase/neon/pull/4603
+
+          FAILED=false
+          for postgres in echo "postgres-v14 postgres-v15"; do
+            expected=$(cat vendor/revisions.json | jq --raw-output '."'"${postgres}"'"')
+            actual=$(git rev-parse "HEAD:vendor/${postgres}")
+            if [ ${expected} != ${actual} ]; then
```

```suggestion
            if [ "${expected}" != "${actual}" ]; then
```

Let's go with quotes here as these are strings, which we hope to be commits, but could be other ones as well.